app/assets/stylesheets/bootstrap_overrides.scss

```diff
@@ -88,6 +88,12 @@ a {
   font-weight: bold;
   background-color: inherit;
 }
+.navbar-default .navbar-nav > .open > a,
+.navbar-default .navbar-nav > .open > a:hover,
+.navbar-default .navbar-nav > .open > a:focus {
```

Comment:
something to note - when possible, use specific classes on elements for css. CSS is parse right-to-left, so if your selector is just a tag name - i.e., `a` - then the CSS parser will first find all anchor tags in the *entire* document, then scope them based on `.open`, etc.

Not an issue in a small-to-medium sized app, but something to be aware of.